[OData V2] Document function import calls in batch change sets (#261)
SAP/cloud-sdk: docs/java/features/odata/type-safe-client-odata-v2.mdx

The `service` object offers the method `batch` which allows access to the batch-related methods that help build the batch request.

> Multiple single requests (like `createBusinessPartnerAddress`) can be wrapped into so-called changesets. 

Reviewer: ```suggestion
Multiple single requests like `createBusinessPartnerAddress` and others can be wrapped into an [OData BATCH  ChangeSet](https://www.odata.org/documentation/odata-version-2-0/batch-processing/). 
```

> Changesets are meant to wrap transactional requests together.

Reviewer: ```suggestion
OData BATCH ChangeSets are meant to wrap transactional requests together for better performance.
```

> Additionally, changesets can also contain function import calls (use the method `addFunctionImport(...)`).

Reviewer: ```suggestion
OData Function Import can be also used inside of OData Batch ChangeSets, for that use the method `addFunctionImport(...)`.
```
